docs/development.md

```diff
@@ -40,7 +40,8 @@ After doing so, you can run Nativefier with your test parameters:
 nativefier --your-awesome-new-flag 'https://your-test-site.com'
 ```
 
-Then run your nativefier app *through the command line too* (to see logs & errors):
+Then run your nativefier app _through the command line too_ (to see logs & errors):
+
```

Comment:
I think Prettier did this - not sure if it matters

Reply:
(I'll revert if needed)

Reply:
No worry, if Prettier prefers this, whatever 🙂